pandas/core/dtypes/dtypes.py

```diff
@@ -701,20 +721,20 @@ def tz(self):
         return self._tz
 
     @classmethod
-    def construct_array_type(cls):
+    def construct_array_type(cls) -> Type["DatetimeArray"]:
         """
         Return the array type associated with this dtype.
 
         Returns
         -------
         type
         """
-        from pandas.core.arrays import DatetimeArray
+        from pandas.core.arrays import DatetimeArray  # noqa: F811
```

Comment:
Why is this needed now?

Reply:
redefinition of unused 'DatetimeArray' from line 29 flake8(F811)